File: 03- Subprogramas y recursividad/subprogramas/java/excepciones/LanzarExcepciones.java
/**
 * Descripción: Lanzamiento de excepciones.
 * Autor: Santiago Nicolás De la mora Núñez
 * Fecha: 25/02/2026
 */

public class LanzarExcepciones {
    public static void main(String[] args) {
        int edad = -5;

        if (edad < 0) {
            throw new IllegalArgumentException("La edad no puede ser negativa");
        }

        System.out.println("Edad válida");
    }
}

File: 03- Subprogramas y recursividad/subprogramas/python/excepciones/lanzar_excepciones.py
"""
Descripción: Lanzamiento de excepciones.
Autor: Santiago Nicolás De la mora Núñez
Fecha de creación: 10/04/2026
"""

edad = -5

if edad < 0:
    raise ValueError("La edad no puede ser negativa")

print("Edad válida")